### PR TITLE
Add Content

### PR DESCRIPTION
**Pull request by tjb1019**
Replaced the boilerplate jumbotron with our new jumbotron which contains the four categories of poems we will be offering.  Users will ultimately be able to click those links and be taken to a page that has poems of that category.

I also added three "poem summaries" in the section beneath that.  Each summary shows the poem title, author, category, rating, and a clickable icon that will allow users to save their favorite poems.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,29 +55,73 @@
     <!-- Main jumbotron for a primary marketing message or call to action -->
     <div class="jumbotron">
       <div class="container">
-        <h1>Hello, world!</h1>
-        <p>This is a template for a simple marketing or informational website. It includes a large callout called a jumbotron and three supporting pieces of content. Use it as a starting point to create something more unique.</p>
-        <p><a class="btn btn-primary btn-lg" href="#" role="button">Learn more &raquo;</a></p>
+        <div class="row">
+          <h1 class="text-center">Popular Categories</h1>
```
Comment (Kimmyrae) [inline index.html:59]: Change "Popular Categories" to "Categories".

File: index.html [hunk 2]
```diff
@@ -55,29 +55,73 @@
...
+            <a href="#" class="thumbnail">
+              <img src="images/placeholder.png" alt="placeholder image">
+            </a>
+          </div>
+        </div>
       </div>
     </div>
 
     <div class="container">
       <!-- Example row of columns -->
```
Comment (Kimmyrae) [inline index.html:87]: remove comment

**Comment by Kimmyrae** [comment]
The code is super clean and organized, much how I aspire to write my own code. It was difficult to find anything to fix.

File: index.html [hunk 3]
```diff
@@ -55,29 +55,73 @@
...
+        <h1 class="text-center">New &amp; Popular</h1>
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object" src="images/placeholder-200.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">Happy Valley</h4>
+            <p class="poem-info"><span class="author">Alice Lyons</span><span class="category">Nature</span><span class="rating">&#9734;&#9734;&#9734;&#9734;</span><span class="add-favorite pull-right glyphicon glyphicon-plus"></span></p>
```
Comment (katiecann) [inline index.html:98]: We need additional spacing between the "author," "category," and "rating" classes.

File: index.html [hunk 4]
```diff
@@ -55,29 +55,73 @@
...
+        </div>
+        <div class="media">
+          <div class="media-left">
+            <a href="#">
+              <img class="media-object" src="images/placeholder-200.png" alt="placeholder image">
+            </a>
+          </div>
+          <div class="media-body">
+            <h4 class="media-heading">And Then It Was Less Bleak Because We Said So</h4>
+            <p class="poem-info"><span class="author">Wendy Xu</span><span class="category">Love</span><span class="rating">&#9734;&#9734;</span><span class="add-favorite pull-right glyphicon glyphicon-plus"></span></p>
```
Comment (katiecann) [inline index.html:110]: In need of separator lines above and below the "poem-info" class to separate it visually from the rest of the poem data.

**Pull request closed without merging**